EveripediaNetwork/ep-api
# relayer service

# Implements relayer service
_The relayer module receives a splitted transaction and then forwards it to the wiki contract _

## How should this be tested?
1. yarn dev
2. Send a transaction from the client with the required params and then see the result in the block explorer

## Linked issues
closes https://github.com/EveripediaNetwork/issues/issues/110
